run.go

```diff
@@ -55,6 +69,14 @@ function onerror(msg, src, line, col, err) {
 }
 `
 
+const inlineTemplate = `
+import { log, write, read } from '@jkcfg/std';
```

Comment:
Huum, also `Format`, `Encoding`, ... should we do `import * as std from '@jkcfg/std/std'` instead (or in addition to?) We'd be future-proof if we use `*`.

Reply:
Yeah good point -- let's adjust in another PR (or refactor)